tests/integration/test_array_creation.py

```diff
@@ -189,8 +189,8 @@ def test_full_like_assertion():
     (3.5,),
     (2, 10),
     (-2.5, 10.0),
-    ### output: num: array([ 1, -1, -3, -5, -7]),
-    ### np: array([ 1. , -1.5, -4. , -6.5, -9. ]
+    # output: num: array([ 1, -1, -3, -5, -7]),
+    # np: array([ 1. , -1.5, -4. , -6.5, -9. ]
     # (1, -10, -2.5),
```

Comment:
@magnatelee Is this difference in results between numpy and cunumeric here expected or a bug?